rustls/src/conn/unbuffered.rs

```diff
@@ -170,6 +170,7 @@ impl<Data> UnbufferedConnectionCommon<Data> {
 }
 
 /// The current status of the `UnbufferedConnection*`
+#[allow(clippy::exhaustive_structs)]
```

Comment:
Is this externally constructed? Not convinced it's definitionally complete. Same for the other `unbuffered` types.